submissions/krylenger/a-tiny-JS-world_post_OOP/index.js

```diff
@@ -0,0 +1,140 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details */
+
+const dog = {
+    type: "animal",
+    legs: 4,
+    hands: 0,
+    gender: "male",
+    friends: ["Anna", "Eugene"],
+  },
+  cat = {
+    type: "animal",
```

Comment:
Task requires to expose species of each inhabitant. "animal" is not a species.

Please apply the same to all classes.

Reply:
✅ Done